iz_importer.py

```diff
@@ -57,9 +59,11 @@ def __init__(self):
 
         print("Starting to process loaded core files...")
         self.process_loaded_files()
-
-        self.monitoring_tools.send_monitoring_report(subject=f"IZ_BATCH:{get_pst_time_now_string()}",
-                                                     time_stamp=starting_time_stamp)
+        if platform.system() != 'Darwin':
```

Comment:
Just and FYI, I have a parallel change to this line on my branch, it is ok because we can resolve the merge conflict when the time comes. On my branch I changed monitoring reports to only be sent if the len(mailing_list) > 0 so that sending an email will be optional.

Reply:
I like your version better; I'll back mine out.